modules/aws/files/permission_boundary_iam_policy.json.tpl

```diff
@@ -146,13 +146,13 @@
         "iam:Set*"
       ],
       "Resource": [
-        "arn:${partition}:iam:::policy/StreamNative/StreamNativeCloudBootstrapPolicy",
-        "arn:${partition}:iam:::policy/StreamNative/StreamNativeCloudLBPolicy",
-        "arn:${partition}:iam:::policy/StreamNative/StreamNativeCloudManagementPolicy",
-        "arn:${partition}:iam:::policy/StreamNative/StreamNativeCloudPermissionBoundary",
-        "arn:${partition}:iam:::policy/StreamNative/StreamNativeCloudRuntimePolicy",
-        "arn:${partition}:iam::${account_id}:role/StreamNative/StreamNativeBootstrapRole",
-        "arn:${partition}:iam::${account_id}:role/StreamNative/StreamNativeManagementRole"
+        "arn:${partition}:iam::${account_id}:policy/StreamNative/StreamNativeCloudBootstrapPolicy",
+        "arn:${partition}:iam::${account_id}:policy/StreamNative/StreamNativeCloudLBPolicy",
+        "arn:${partition}:iam::${account_id}:policy/StreamNative/StreamNativeCloudManagementPolicy",
+        "arn:${partition}:iam::${account_id}:policy/StreamNative/StreamNativeCloudPermissionBoundary",
+        "arn:${partition}:iam::${account_id}:policy/StreamNative/StreamNativeCloudRuntimePolicy",
+        "arn:${partition}:iam::${account_id}:role/StreamNative/StreamNativeCloudBootstrapRole",
+        "arn:${partition}:iam::${account_id}:role/StreamNative/StreamNativeCloudManagementRole"
       ]
```

Comment:
Looking at https://github.com/search?q=org%3Astreamnative+StreamNativeBootstrapRole&type=code,

perhaps you want to change https://github.com/streamnative/terraform-managed-cloud/blob/main/examples/aws/policy_files/permission_boundary_policy.json and https://github.com/streamnative/terraform-managed-cloud/blob/main/modules/aws/README.md (note, in the readme it is in a collapsible summary).

Reply:
I only proposed the change here as Mao redirected me, but I'm totally not knowing what I'm doing.

So I leave it to @jdmaguire and @gengmao. Just trying to help here

Reply:
Thanks @AlvaroStream. @jdmaguire meant there are other [places](https://github.com/search?q=repo%3Astreamnative%2Fterraform-managed-cloud%20StreamNativeBootstrapRole&type=code) used `StreamNativeBootstrapRole` wrong: it should be `StreamNativeCloudBootstrapRole`. Appreciate it you can update them in this PR too.

Reply:
Done